Core/CoreUITests/CoreUITestCase.swift

```diff
@@ -185,7 +185,7 @@ open class CoreUITestCase: XCTestCase {
         if data.isEmpty {
             return nil
         } else {
-            return (try? JSONDecoder().decode(LoginSession?.self, from: data))!
+            return try? JSONDecoder().decode(LoginSession?.self, from: data)
```

Comment:
I don't like that this can silently fail.
```suggestion
        if data.isEmpty || data == "null".data(using: .utf8) {
            return nil
        } else {
            return (try? JSONDecoder().decode(LoginSession?.self, from: data))!
        }
```

Reply:
Should there always be a currentSession? I had this crash on me which is why I changed it. Maybe something was wrong with the test?

Reply:
No, but if the JSON is corrupt it shouldn't silently fail

Reply:
⬆️ See the suggestion 2 lines up